interpreter/binary/encode.ml

```diff
@@ -141,15 +141,11 @@ struct
     | Immutable -> u8 0
     | Mutable -> u8 1
 
-  let resumability = function
-    | Terminal -> u8 0
-    | Resumable -> u8 1
-
   let global_type = function
     | GlobalType (t, mut) -> value_type t; mutability mut
 
-  let tag_type = function
-    | TagType (ft, res) -> resumability res; func_type ft  (* TODO *)
+  let tag_type (TagType x) =
+    vu32 0x00l; var_type x
```

Comment:
@rossberg I reinstated 0x00 on tags (it is present in the exception handling proposal too).